dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -5661,6 +5661,11 @@ void gp_walk(const Item* item, void* arg)
             {
                 boost::shared_ptr<SimpleColumn> scsp(sc->clone());
                 gwip->scsp = scsp;
+                
+                if (col->type() == Item::FIELD_ITEM)
+                {
+                    gwip->columnMap.insert(CalpontSelectExecutionPlan::ColumnMap::value_type(string(((Item_field*)item)->field_name.str), scsp));
```

Comment:
Fixed issue by inserting new column mapping for case Item::Ref_Item when coltype is Field_Item (this happens when it's column of view).

Reply:
Looks OK to me, but I don't know this code very well, so lets wait for Roman's input before merging.

Reply:
Looks OK for me too. However we need to run full regression with this oneliner.
I just have aestetic request for now. Could you make break this oneliner to make it less then 80 symbols to make it readable in vim with multiply panes? Here and in 1.2 PR also.